recvalsite/settings.py

```diff
@@ -106,10 +106,14 @@
     "RECVAL_SQLITE_DB_PATH", default=BASE_DIR / "db.sqlite3", cast=Path
 )
 
+USE_TEST_DB = config("USE_TEST_DB", default=False)
+
```

Comment:
Normally Django’s unit tests use a temporary in-memory database that is quite fast. We only `USE_TEST_DB` in morphodict because the import process is too slow to load the required test data for every test. And it’s a bit of a pain.

If there isn’t a specific need to have a bunch of data that’s time-consuming to compute in the test database, I would recommend entirely skipping anything to do with `USE_TEST_DB`.

(If this is meant for cypress, I have other suggestions for that, but I don’t think that’s in scope for this PR.)

```suggestion
```

Reply:
`USE_TEST_DB` was initially included so we could load specific entries into the DB and look for them in the tests. We don't actually use it in the tests anymore, but it seemed like a fine variable to have.

Reply:
Fair enough, it doesn't really make sense to have something in there "just in case".